data-serving/README.md

```diff
@@ -0,0 +1,115 @@
+  which sends and receives case objects via the aforementioned curator API. So again, if something changes in the API,
+  it will affect the model objects used in the UI.
+* The
+  **[CSV → json converter](https://github.com/globaldothealth/list/blob/175bbdd66e0cc02a87629969b70658783fe75518/data-serving/scripts/convert-data/convert_data.py)**,
+  which converts the existing (Sheets-originated) CSV data to json that conforms to the aforementioned MongoDB schema.
+  If you add a _new_ field to the case schema that is not present in the old data, you don't need to worry about this;
+  however, if you're modifying a field that is part of the conversion process, the converter will need to be updated to
+  generate the correct fields/data.
+* The **mongodb → CSV exporter**, which exports specified fields from the MongoDB cases into a CSV format that we can
+  make available to researchers, similar to the CSV that was generated from Google Sheets originally. If you add,
+  remove, or rename a field and it's part of (or should be added to) the CSV export, you'll need to update the exporter.
+  [TBD/WIP]
+* The **sample data**, which unfortunately is sprinkled throughout the stack, and is used for seeding local databases
+  and for unit & integration testing. The examples need to be updated with the changes.
+    * [Seeding local databases](https://github.com/globaldothealth/list/tree/e85dacd056db71e1d7d4b56034536ccc6dbee23c/data-serving/samples)
+    * [Dataserver unit test fixtures](https://github.com/globaldothealth/list/tree/e85dacd056db71e1d7d4b56034536ccc6dbee23c/data-serving/data-service/test/model/data)
+    * Integration test fixtures
+      [[1](https://github.com/globaldothealth/list/blob/224abab6189af6e5a0ad26ae082167c2a3f5966a/verification/curator-service/ui/cypress/support/commands.ts)],
+      [[2](https://github.com/globaldothealth/list/blob/ca177c1b9a940dcd0f51166db6c6f13976894dda/verification/curator-service/ui/src/components/fixtures/fullCase.json)]
+
+#### Sample PRs
+
+* [Adding a field to the schema](https://github.com/globaldothealth/list/pull/318/files)
+* [Modify/rename a field](https://github.com/globaldothealth/list/pull/372/files)
+* [Remove a field](https://github.com/globaldothealth/list/pull/257/files)
+
+### Testing a schema update
+
+1. First, make the changes to the affected parts of the stack, including sample data and test fixtures.
+1. Run
+   [dev/setup\_db.sh](https://github.com/globaldothealth/list/blob/ca177c1b9a940dcd0f51166db6c6f13976894dda/dev/setup_db.sh)
+   and check the output for errors. If one or more documents fails to import, there may be a mismatch between your
+   [sample data](https://github.com/globaldothealth/list/tree/e85dacd056db71e1d7d4b56034536ccc6dbee23c/data-serving/samples)
+   and your
+   [MongoDB json schema](https://github.com/globaldothealth/list/blob/ca177c1b9a940dcd0f51166db6c6f13976894dda/data-serving/data-service/schemas/cases.schema.json)
+   is.
```

Comment:
Couple extra `is`'s here and on line 72.

Reply:
What is is is the problem with that? is.